src/components/BlockTitleDescription/index.js

```diff
@@ -25,4 +26,28 @@ class BlockTitleDescription extends PureComponent {
   }
 }
 
+BlockTitleDescription.propTypes = {
+  block: PropTypes.shape({
+    fields: PropTypes.shape({
+      richText: PropTypes.shape({
+        nodeType: PropTypes.string,
+        data: PropTypes.object,
+        content: PropTypes.array
+      })
+    })
+  })
+};
+
+BlockTitleDescription.defaultProps = {
+  block: {
+    fields: {
+      richText: {
+        nodeType: '',
+        data: {},
```

Comment:
should this be `null` instead of `{}`?

Reply:
When there is no data in ```data``` field, Contentful is sending an empty object and I am using Contentful extension to parse through ```richText``` so I think it is better to keep as default as possible would be the best.